network.go

```diff
@@ -250,7 +255,7 @@ func (net *Network) ConnectHemispheres() {
                 excitatory = true
             }
             toNodes[node] = &ConnInfo{
-                Strength: RandFloat(0.75, 1.75),
+                Strength: RandFloat(0.75, 1.75), // magic numbers
```

Comment:
I mean I would but I don't know what any of the specific variable names should be; plus I'd imagine it'd be best with short comments describing each variable, and of course I wouldn't know what to put in those comments (I didn't program this nor do I know much about neural networks!).